### PR TITLE
Okcomputer tweaks

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: config/initializers/okcomputer.rb [hunk 1]
```diff
@@ -61,9 +62,7 @@ def profile
 OkComputer::Registry.register 'suri_url', OkComputer::HttpCheck.new(Settings.SURI.URL)
 
 # Stacks
-OkComputer::Registry.register 'stacks_doc_cache_storage_root', OkComputer::DirectoryCheck.new(Settings.STACKS.DOCUMENT_CACHE_STORAGE_ROOT)
```
Comment (ndushay) [inline config/initializers/okcomputer.rb]: so are we saying that these aren't on -dev environment, so they're not worth checking?

Comment (cbeer) [inline config/initializers/okcomputer.rb]: They aren't on -dev, -stage, or -prod.

Comment (ndushay) [inline config/initializers/okcomputer.rb]: I wonder if it's for running locally?

Comment (ndushay) [inline config/initializers/okcomputer.rb]: If these settings are not used, make 'em disappear altogether??

Comment (cbeer) [inline config/initializers/okcomputer.rb]: Ticketed: https://github.com/sul-dlss/argo/issues/874

Comment (jmartin-sul) [inline config/initializers/okcomputer.rb]: IIRC (and i may be misremembering, it's been a while):  file content retrieval is now proxied through dor-services-app, and argo no longer does the SFTP thing.

**Pull request merged**